pkg/client/daemon/session.go

```diff
@@ -442,6 +446,10 @@ func (s *session) run(c context.Context) error {
 }
 
 func (s *session) stop(c context.Context) {
+	s.scout.Report(c, "incluster_dns_queries",
```

Comment:
This looks good. I think if we notice we have huge delays with getting these metrics bc sessions are long, we could always send updates on some cadence (every hour or so).. but I agree that that's more complicated and we should stick to the simpler solution unless we ~ need ~ the more involved one.